modules/phase_field/src/base/PhaseFieldApp.C

```diff
@@ -99,6 +99,7 @@ PhaseFieldApp::associateSyntax(Syntax & syntax, ActionFactory & action_factory)
   registerSyntax("PolycrystalVoronoiVoidICAction", "ICs/PolycrystalICs/PolycrystalVoronoiVoidIC");
   registerSyntax("RigidBodyMultiKernelAction", "Kernels/RigidBodyMultiKernel");
   registerSyntax("Tricrystal2CircleGrainsICAction", "ICs/PolycrystalICs/Tricrystal2CircleGrainsIC");
+  registerSyntax("GPMKernelAction", "Kernels/GPMKernels");
```

Comment:
I'd perefer the new syntax path convention 

Modules/PhaseField/GrandPotential

Let's avoid user-visible acronyms